central/telemetry/centralclient/interceptors.go

```diff
@@ -7,15 +7,36 @@ import (
 	"github.com/stackrox/rox/pkg/telemetry/phonehome"
 )
 
+const snowIntegrationHeader = "Rh-ServiceNow-Integration"
+const anyValueOrMissing = ""
+
 var (
-	trackedPaths []string
 	ignoredPaths = []string{"/v1/ping", "/v1.PingService/Ping", "/v1/metadata", "/static/*"}
```

Comment:
Note for the reviewers: `ignoredPaths` is not part of the campaign configuration, but a hardcoded generic condition.